### PR TITLE
add

### PR DESCRIPTION
**Pull request by mdoam**

### CODE REVIEW

File: CartApi/Infrastructure/ActionResults/InternalServerErrorObjectResult.cs [hunk 1]
```diff
@@ -0,0 +1,14 @@
+﻿using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+
+namespace ShoesOnContainers.Services.CartApi.Infrastructure.ActionResults
```
Comment (vlau26) [inline CartApi/Infrastructure/ActionResults/InternalServerErrorObjectResult.cs]:
```suggestion
namespace EventBriteAssignment.Services.CartApi.Infrastructure.ActionResults
```
change namespace name, should match with Rebka's namespace name change too

File: CartApi/Infrastructure/Exceptions/CartDomainException.cs [hunk 2]
```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace ShoesOnContainers.Services.CartApi.Infrastructure.Exceptions
```
Comment (vlau26) [inline CartApi/Infrastructure/Exceptions/CartDomainException.cs]:
```suggestion
namespace EventBriteAssignment.Services.CartApi.Infrastructure.Exceptions
```
change namespace name

File: CartApi/Infrastructure/Filters/HttpGlobalExceptionFilter.cs [hunk 3]
```diff
@@ -0,0 +1,64 @@
+﻿
+using ShoesOnContainers.Services.CartApi.Infrastructure.ActionResults;
```
Comment (vlau26) [inline CartApi/Infrastructure/Filters/HttpGlobalExceptionFilter.cs]: once namespace names are changed, update here to include the appropriate namespace. Lines 2, 3 and line 10

File: CartApi/Messaging/Consumers/OrderCompletedEventConsumer.cs [hunk 4]
```diff
@@ -0,0 +1,30 @@
+﻿using Common.Messaging;
+using MassTransit;
+using Microsoft.Extensions.Logging;
+using ShoesOnContainers.Services.CartApi.Model;
```
Comment (vlau26) [inline CartApi/Messaging/Consumers/OrderCompletedEventConsumer.cs]: namespace name change

File: CartApi/Model/Cart.cs [hunk 5]
```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace ShoesOnContainers.Services.CartApi.Model
```
Comment (vlau26) [inline CartApi/Model/Cart.cs]: namespace name change

File: CartApi/Model/Cart.cs [hunk 6]
```diff
@@ -0,0 +1,19 @@
...
+{
+    public class  Cart
+    {
+        public string BuyerId { get;  set; }
+        public List<CartItem> Events { get; set; } 
+
+        public Cart(string cartId)
+        {
+            BuyerId = cartId;
+            Events = new List<Model.CartItem>();
```
Comment (vlau26) [inline CartApi/Model/Cart.cs]:
```suggestion
            Events = new List<CartItem>();
```
I believe 'Model. ' is not needed if namespace is correct

File: CartApi/Model/CartItem.cs [hunk 7]
```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace ShoesOnContainers.Services.CartApi.Model
```
Comment (vlau26) [inline CartApi/Model/CartItem.cs]: update namespace name

**Review by vlau26 — CHANGES_REQUESTED** [review]
minor changes regarding namespace names only. 👍  then for me it's ready to merge !

**Review by mdoam — COMMENTED** [review]
Hi all, I've changed namespace to appropriate name. Let me know if the codes are ready to merge.

**Review by vlau26 — APPROVED** [review]
I'm good with you merging it after Rebka or Sunny reviews and approves. 👍

**Comment by parks38** [comment]
The code seems good to merge!

**Pull request merged**